keyboards/ogurec/keymaps/dack/readme.md

```diff
@@ -0,0 +1,3 @@
+# Dack's Ogurec Keymap
+
+This is Dack's personal ogurec keymap and it is for left-sided PM placement
```

Comment:
Just by the way, if you want to make this keymap unavailable to build for `right_pm` you can move it to another `keymaps/` folder under `left_pm`.